### ISSUE TITLE
Add `Click Element When Visible` to RPA.Browser library

### ISSUE THREAD

**Issue by janipalsamaki** [robocorp/rpaframework#38]
We already have `Input Text When Element Is Visible`. Could we also have `Click Element When Visible`? Almost all web robots need that one since more and more content is loaded dynamically.

For example:

```robot
Accept cookie consent
    ${accept_cookies_button}=    Set Variable    css:.browser-consent button.accept
    Wait Until Element Is Visible    ${accept_cookies_button}
    Click Button    ${accept_cookies_button}
```

```robot
Accept cookie consent
    Click Element When Visible    css:.browser-consent button.accept
```

**Comment by osrjv** [comment]
A keyword named `Wait and click button` already exists but I'll add your suggestion as an alias to be more in line with other keywords.

**Comment by janipalsamaki** [comment]
Just to check: Is there something button-only-specific in the `Wait And Click Button`? The idea for the new `Click Element When Visible` keyword would be to click on _any_ element that matches the given locator, not just buttons.

**Comment by osrjv** [comment]
You're right, it does use the specific `Click Button` keyword under the hood. In the next release I'll add two new keywords: `Click Element When Visible` and `Click Button When Visible`, shadowing the corresponding keywords in Selenium library.

**Comment by osrjv** [comment]
The keywords mentioned in the previous comment are now available in release `2.5.0`.
